js/anbox-stream-sdk.js

```diff
@@ -4041,11 +4046,13 @@ class AnboxVhalManager {
   /**
    * Constructor to initialize a new instance of the AnboxVhalManager which is
    * responsible to communicate with an Android VHAL.
-   * @param webrtcManager {object} Anbox WebRTCManager object
+   * @param webrtcManager {object} Anbox WebRTCManager object.
+   * @param vhalReady {function} Callback to notify subscribers that we have received VHAL prop configs.
```

Comment:
```suggestion
   * @param vhalReady {function} Callback to notify subscribers that we have received VHAL prop configs.
```
> that we have received VHAL prop configs.

Isn't that an implementation detail, though? I guess what subscribers really care about is to be notified that the VHAL is ready, how this is done internally is probably not relevant to them.

Reply:
Not sure what you are suggesting to change here? Is the JS doc comment (to something like "that VHAL is ready") or the actual way it is done? :thinking:

Reply:
Yes, it was just about the docs, thanks for changing it!